prow/plugins/dco/dco.go

```diff
@@ -97,6 +117,32 @@ type commentPruner interface {
 	PruneComments(shouldPrune func(github.IssueComment) bool)
 }
 
+// checkTrustedUser checks are all commits from a trusted user
+func checkTrustedUser(dco plugins.Dco, gc gitHubClient, l *logrus.Entry, org, repo string, number int) (bool, error) {
+	// Skip checking is user trusted if this option isn't enabled.
+	if !dco.SkipDcoCheck {
```

Comment:
nit: if you move this outside this function (in `handle`), you can avoid passing the dco config and reduce a function call

Reply:
Fixed